pyglet/image/__init__.py

```diff
@@ -1359,14 +1417,35 @@ def blit(self, x, y, z=0, width=None, height=None):
 
         glBindTexture(self.target, 0)
 
-    def blit_into(self, source, x, y, z):
+    def blit_into(self, source: ImageData, x: int, y: int, z: int) -> None:
+        """
+        Draw `source` on this image.
+
+        `source` will be copied into this image such that its anchor point
+        is aligned with the `x` and `y` parameters.  If this image is a 3D
+        texture, the `z` coordinate gives the image slice to copy into.
+
+        Note that if `source` is larger than this image (or the positioning
+        would cause the copy to go out of bounds) then you must pass a
+        region of `source` to this method, typically using get_region().
+
+        :Parameters:
```

Comment:
This can be removed. It's also incorrect - (x, y, z) do not mean the anchor points.